Source/santad/SNTDecisionCacheTest.mm

```diff
@@ -85,8 +85,8 @@ - (void)testResetTimestampForCachedDecision {
 
   OCMExpect([self.mockRuleDatabase
     resetTimestampForRule:[OCMArg checkWithBlock:^BOOL(SNTRule *rule) {
-      return rule.identifier == cd.sha256 && rule.state == SNTRuleStateAllowTransitive &&
-             rule.type == SNTRuleTypeBinary;
+      return [rule.identifier isEqualToString:cd.sha256] &&
```

Comment:
this change is not related to the overall PR, however this test started failing because it was using the wrong equality comparison for ObjC objects.

Not sure how this ever passed previously ???  ¯\\\_(ツ)_/¯

Reply:
That's concerning. We should probably set aside time to make sure our tests are accurate for the next fixit.